django/chat/views.py

```diff
@@ -910,3 +917,120 @@ def update_qa_options_from_librarian(request, chat_id, library_id):
             "trigger_library_change": "true" if library != original_library else None,
         },
     )
+
+
+def generate_prompt(task_or_prompt: str):
+
+    load_dotenv()
+    openai.api_key = os.getenv("AZURE_OPENAI_KEY")
+
+    openai.api_version = os.getenv("AZURE_OPENAI_VERSION")
```

Comment:
You shouldn't need these 3 lines now, or the imports for them, since you are using OttoLLM class.